civihr_default_theme/scss/components/_header.scss

```diff
@@ -150,14 +161,17 @@
 
   @media(min-width: $screen-md-max) {
     a {
-      color: inherit;
+      color: $light-pale;
```

Comment:
Do we have colours for different screen sizes? Why does this have to be placed inside a `@media` block?

Reply:
Mobile menu

<img width="250" alt="mobile-menu" src="https://user-images.githubusercontent.com/6400898/32112271-17f29a38-bb3d-11e7-8d2c-c60066c2b1fa.png">